crates/binstalk/src/errors.rs

```diff
@@ -399,9 +400,9 @@ impl Termination for BinstallError {
     fn report(self) -> ExitCode {
         let code = self.exit_code();
         if let BinstallError::UserAbort = self {
-            writeln!(io::stdout(), "Installation cancelled").ok();
+            info!("Installation cancelled");
```

Comment:
That used to be a warn in the previous iteration, may as well revert back to that.

Reply:
Done!